pkg/confgen/flowlogs2metrics_config.go

```diff
@@ -21,161 +21,163 @@ import (
 	"fmt"
 	"io/ioutil"
 
+	"github.com/netobserv/flowlogs-pipeline/pkg/api"
+	config2 "github.com/netobserv/flowlogs-pipeline/pkg/config"
```

Comment:
I think it's better to reserve the word `config` for this import and rename all the other variables named "config".